README.md

```diff
@@ -17,7 +17,7 @@
 * [FAQ](#faq)
 
 ## Installation
-Supported Python version is >= **3.4**.
+Supported Python version is >= **3.5**.
```

Comment:
Two more places to update:
https://github.com/BayesWitnesses/m2cgen/blob/b8e4fac64e5f9267693c08294adc402b179802c2/setup.py#L28
https://github.com/BayesWitnesses/m2cgen/blob/b8e4fac64e5f9267693c08294adc402b179802c2/setup.py#L35

Reply:
Good catch, thank you